EIPS/eip-3670.md

```diff
@@ -91,49 +133,21 @@ def validate_code(code: bytes):
         # Ensure the opcode is valid
         opcode = code[pos]
         pos += 1
-        assert opcode in valid_opcodes
+        if not opcode in valid_opcodes:
```

Comment:
We should be able to use `assert` with a message directly in python3
```suggestion
        assert (not opcode in valid_opcodes), "undefined instruction"
```

Here and below